pkg/kp/kv.go

```diff
@@ -52,6 +52,26 @@ type WatchResult struct {
 	Status  string
 	Output  string
 	Time    time.Time
+	Expires time.Time `json:"expires,omitempty"`
```

Comment:
I added this field comment so I could specify the "omitempty" option, indicating that this field might not be present. The default JSON marshaller handles the other fields well enough; they don't need special handling.

But I do need a change here. As mentioned in another PR, the JSON field name should be capitalized, `Expires`, to match the other fields.  Interestingly, the YAML marshaller adjusts the case of its fields, and I was expecting that behavior from JSON too.